support/build/Makefile.rules

```diff
@@ -857,7 +867,8 @@ $(call libname2preolib,$(1)): $($(call vprefix_lib,$(1),OBJS)) \
 			      $($(call vprefix_lib,$(1),DTB)) \
 			      $($(call vprefix_lib,$(1),DTB-y))
 	$(call build_cmd,LD,,$(call libname2preolib,$(1)),\
-		$(LD) $(LIBLDFLAGS) $(LIBLDFLAGS-y) \
+		$(LD),\
```

Comment:
Hi,
There seems to be a typo here, what's with the extra `,` ?

Reply:
The comma seperates the `$command` from the new optional `$longargs` argument. Everything in longargs will be written into the built_args file if the the config option `BUILD_ARGS_FILE` is enabled and replaced with a reference to that file.

Reply:
Makes sense now, thanks!!